encryption.md

```diff
@@ -0,0 +1,48 @@
+
+
+### Sender Authenticated Encryption
+
+When a sender would like to encrypt a message for a recipient or multiple recipients and also be authenticated such that the recipients can determine they were the party that encrypted the message, the JWA of `ECDH-1PU` defined by [draft](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-02) SHOULD be used within the structure of a JWE. 
+
+For the keys involved in key agreement, the following elliptic curves MUST be supported.
+
+|Curve |Description                                                                                                                        |
+|------|-----------------------------------------------------------------------------------------------------------------------------------|
+|X25519|The underlying curve is actually `Curve25519`, however when used in the context of diffie helman the identifier of `X25519` is used|
+|P-256 |NIST defined P-256 elliptic curve                                                                                                  |
+
+For content encryption of the message, the following algorithms MUST be supported.
+
+|Algorithm(JWA) |Description                                          |
+|---------------|-----------------------------------------------------|
+|C20P           |ChaCha20Poly1305                                     |
+|AES-GCM        |Advanced Encryption Standard with galois/counter mode|
+
+### Anonymous Encryption
+
+When a sender would like to encrypt a message for a recipient or multiple recipients but not be authenticated by the recipients as the party who encrypted the message, the JWA of `ECDH-ES` defined by [RFC 7518](https://tools.ietf.org/html/rfc7518#section-4.6) SHOULD be used within the structure of a JWE. 
+
+For the keys involved in key agreement, the following elliptic curves MUST be supported.
+
+|Curve |Description                                                                                                                        |
+|------|-----------------------------------------------------------------------------------------------------------------------------------|
+|X25519|The underlying curve is actually `Curve25519`, however when used in the context of diffie helman the identifier of `X25519` is used|
+|P-256 |NIST defined P-256 elliptic curve                                                                                                  |
+
+For content encryption of the message, the following algorithms MUST be supported.
+
+|Algorithm(JWA) |Description                                          |
+|---------------|-----------------------------------------------------|
+|C20P           |ChaCha20Poly1305                                     |
```

Comment:
XC20P here with the definition being XChaCha20Poly1305

Reply:
fixed.